templates/service/project/AppDependencies.scala

```diff
@@ -1,29 +1,34 @@
-import sbt._
-import play.sbt.PlayImport._
 import play.core.PlayVersion
-import uk.gov.hmrc.SbtAutoBuildPlugin
-import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
-import uk.gov.hmrc.versioning.SbtGitVersioning
-
-object FrontendBuild extends Build with MicroService {
+import play.sbt.PlayImport._
+import sbt.Keys.libraryDependencies
+import sbt._
 
-  val appName = "$!APP_NAME!$"
+object AppDependencies {
 
-  override lazy val appDependencies: Seq[ModuleID] = compile ++ test()
+  def appDependencies: Seq[Setting[_]] = Seq(
+    libraryDependencies ++= compile ++ test()
+  )
 
   val compile = Seq(
+    <!--(if type=="FRONTEND")-->
     "uk.gov.hmrc" %% "govuk-template" % "$!govukTemplateVersion!$",
     "uk.gov.hmrc" %% "play-ui" % "$!playUiVersion!$",
-
+    <!--(end)-->
+    <!--(if MONGO)-->
+    "uk.gov.hmrc" %% "play-reactivemongo" % "$!playReactivemongoVersion!$",
+    <!--(end)-->
     ws,
     "uk.gov.hmrc" %% "bootstrap-play-25" % "$!bootstrapPlay25Version!$"
   )
 
   def test(scope: String = "test") = Seq(
     "uk.gov.hmrc" %% "hmrctest" % "$!hmrcTestVersion!$" % scope,
-    "org.scalatest" %% "scalatest" % "3.0.0" % scope,
+    "org.scalatest" %% "scalatest" % "3.0.4" % scope,
```

Comment:
3.0.5 is out now, I picked 3.0.4 because that's the version I'm using and I didn't want to foist a version I hadn't battle tested myself on others.